var/spack/repos/builtin/packages/py-xarray/package.py

```diff
@@ -63,3 +63,4 @@ class PyXarray(PythonPackage):
     depends_on('py-rasterio', when='+io', type=('build', 'run'))
     depends_on('py-cfgrib',   when='+io', type=('build', 'run'))
     depends_on('py-pooch',    when='+io', type=('build', 'run'))
+    depends_on('py-dask',     when='+io', type=('build', 'run'))
```

Comment:
Dask isn't needed for I/O, it's needed for parallel support. So two changes are required here. First, you'll want to add a `+parallel` variant above:
```python
variant('parallel', default=False, description='Build parallel backend')
```
Then you'll want to change this line to the following (equivalent to `dask[complete]`):
```suggestion
    depends_on('py-dask+array+bag+dataframe+distributed+diagnostics+delayed', when='+parallel', type=('build', 'run'))
```